config/initializers/warden.rb

```diff
@@ -6,19 +6,24 @@
 require 'open_project/authentication/strategies/warden/user_basic_auth'
 require 'open_project/authentication/strategies/warden/session'
 
-strategies = {
-  basic_auth_failure: OpenProject::Authentication::Strategies::Warden::BasicAuthFailure,
-  global_basic_auth:  OpenProject::Authentication::Strategies::Warden::GlobalBasicAuth,
-  user_basic_auth:    OpenProject::Authentication::Strategies::Warden::UserBasicAuth,
-  session:            OpenProject::Authentication::Strategies::Warden::Session
-}
+WS = OpenProject::Authentication::Strategies::Warden
+
+strategies = [
+  [:basic_auth_failure, WS::BasicAuthFailure, 'Basic'],
+  [:global_basic_auth,  WS::GlobalBasicAuth,  'Basic'],
+  [:user_basic_auth,    WS::UserBasicAuth,    'Basic'],
```

Comment:
When sorting in the work package view while logged out I do get the basic auth prompt as the request to the experimental API does not include the `X-Authentication-Scheme` header. Any ideas as to why that stuff in `applicaction.js` does not affect it?